packages/zapier-platform-cli/src/commands/init.js

```diff
@@ -4,11 +4,7 @@ const constants = require('../constants');
 const appTemplates = require('../app-templates');
 
 const init = (context, location) => {
-  context.line('Welcome to the Zapier Platform! :-D');
-  context.line();
-  context.line(constants.ART);
-  context.line();
-  context.line("Let's initialize your app!");
+  context.line('Initializing New App.');
```

Comment:
I like the ASCII art! Any reasons why we want to remove it?

Reply:
From [this comment](https://github.com/zapier/zapier-platform-cli/pull/427#discussion_r282629042), it seemed like we wanted to make this a little more "professional" looking, but I don't have strong feelings about the art (just the `:-D`, which I'm not a fan of). Feel free to re-add if you'd like!